app/models/doc_collection.rb

```diff
@@ -0,0 +1,83 @@
+class DocCollection
+  include ActionView::Helpers::NumberHelper
+
+  attr_accessor :docs
+  attr_reader :size
+
+  def initialize(project, annotator, options, job)
+    @project = project
+    @annotator = annotator
+    @max_text_size = annotator.find_or_define_max_text_size
+    @options = options
+    @job = job
+    @docs = []
+    @size = 0
+  end
+
+  def request_annotate
+    if @docs.length == 1 && @docs.first.body.length > @max_text_size
+      slice_large_document(@docs.first)
+    else
+      hdocs = docs.map{|d| d.hdoc}
```

Comment:
```suggestion
      hdocs = docs.map{_1.hdoc}
```

numbered parameterをつかうとブロック引数を省略できます。
短いブロックの時は省略すると、すっきりします。

Reply:
修正しました。
https://github.com/pubannotation/pubannotation/pull/111/commits/1cb650fed08c658b7ddeb1dda017b1b5b4809b39